package.json

```diff
@@ -12,9 +12,9 @@
   ],
   "main": "src/index.js",
   "scripts": {
-    "test": "npm run lint && npm run test-only",
-    "test-only": "mocha spec/exec.js",
-    "lint": "eslint ./src ./spec",
+    "check": "npm run lint && npm run test",
+    "test": "mocha spec/exec.js",
+    "lint": "eslint src spec",
     "benchmark": "babel-node benchmark/benchmark.js",
```

Comment:
I can double-check to make sure, but I think this is okay as-is.  `babel-node` will transpile the `benchmark.js` source and all of its dependencies when `node` originally starts.  That shouldn't effect benchmark times (since Node is compiling the transpiled bytecode by that point), and the extra memory usage shouldn't be an issue either.

I'll switch the benchmark over to use `React.createElement` just to be sure - if the numbers don't change, I'll leave it as-is.